File: Codechef/Beginner/FLOW016.c
#include <stdio.h>

int gcd(int a, int b)
{
    if (b == 0)
        return a;
    else
        return gcd(b, a % b);
}   

int lcm(int a, int b)
{
    return (a*b)/gcd(a,b);    
}

int main()
{
    int t, a, b;
    scanf("%d", &t);
    while(t--)
    {
        scanf("%d%d", &a, &b);
        printf("%d %d\n", gcd(a,b), lcm(a,b));
    }
    return 0;
}

File: Codechef/Beginner/FLOW016.py
def gcd(a, b):
    if b == 0:
        return a
    else:
        a = a % b
        return gcd(b, a)


def lcm(a, b):
    g = gcd(a, b)
    return ((a * b) / g)

t = input()
while t:
    a, b = raw_input().split()
    a, b = int(a), int(b)
    print "%d %d" % (gcd(a, b), lcm(a, b))
    t -= 1
